### PR TITLE
Remove spaces that have been unreachable for 30 days

### PR DESCRIPTION
**Pull request by gidsi**
We we weren't able to reach these spaces endpoints in at least the last 30 days.

I've double checked by hand, responses are either 404, 5XX or timeouts.

### CODE REVIEW

**Comment by dbrgn** [comment]
Shouldn't we try to contact them first?

At least some of the spaces, e.g. LambdaSpace or Starship Factory seem to be still active.

(On the other hand we can also remove the entries and re-add them later on, in case the endpoints are fixed...)

**Comment by gidsi** [comment]
> Shouldn't we try to contact them first?
> 
> At least some of the spaces, e.g. LambdaSpace or Starship Factory seem to be still active.
> 
> (On the other hand we can also remove the entries and re-add them later on, in case the endpoints are fixed...)

I've tried contacting a lot of them two times now (i might have missed a few, wasn't always easy to find contact information). 

Since we can add them so easily again and because they are gone that long (LambdaSpace and Starship Factory for more then 12 month i think) i would just remove them and add them again.

File: directory.json [hunk 1]
```diff
@@ -162,7 +140,6 @@
   "[hsmr]":"https://hsmr.cc/spaceapi.json",
   "backspace":"https://status.bckspc.de/spacestatus.php",
   "base48":"https://48.io/status/",
-  "brmlab":"http://nat.brmlab.cz/brmd/brmstatus.json",
```
Comment (rnestler) [inline directory.json:165]: I visited this space a few years ago :) Did you contact them?

Comment (gidsi) [reply directory.json:165]: The website doesn't ring a bell right now, might have been down during that time. Sounds like you have a more direct contact possibility :)

**Pull request merged**